web/services/migrations/0006_slack_filter.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.11 on 2020-03-19 12:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
I assume I am misunderstanding something. The model has no changes to the fields so I wasn't expecting a migration?